src/test/java/com/google/step/youtube/YouTubeClientMocks.java

```diff
@@ -0,0 +1,146 @@
+package com.google.step.youtube;
+
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.YouTube.Channels;
+import com.google.api.services.youtube.YouTube.PlaylistItems;
+import com.google.api.services.youtube.YouTube.Search;
+import com.google.api.services.youtube.YouTube.Videos;
+import com.google.api.services.youtube.model.Channel;
+import com.google.api.services.youtube.model.ChannelContentDetails;
+import com.google.api.services.youtube.model.ChannelContentDetails.RelatedPlaylists;
+import com.google.api.services.youtube.model.ChannelListResponse;
+import com.google.api.services.youtube.model.PlaylistItem;
+import com.google.api.services.youtube.model.PlaylistItemListResponse;
+import com.google.api.services.youtube.model.PlaylistItemSnippet;
+import com.google.api.services.youtube.model.ResourceId;
+import com.google.api.services.youtube.model.SearchListResponse;
+import com.google.api.services.youtube.model.SearchResult;
+import com.google.api.services.youtube.model.Video;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.VideoSnippet;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+
+public class YouTubeClientMocks {
+
+    private static Channels.List mockListChannels;
```

Comment:
hmm, you still have static state.  let's GVC to talk about strategies on how you can get rid of it.  I have meetings until 5pm unfortunately.  Let's talk next week.